ietf/utils/jstest.py

```diff
@@ -30,7 +30,7 @@
     print("     "+skip_message)
 
 def start_web_driver():
-    service = Service(log_output=f"{executable_name}.log", service_args=['--log-no-truncate'])
+    service = Service(executable_path=f"/usr/bin/geckodriver", log_output=f"{executable_name}.log", service_args=['--log-no-truncate'])
```

Comment:
Since we already pulled out `executable_name`, we should use it here. Maybe pull out `executable_path` too.

Reply:
```suggestion
    service = Service(executable_path=f"/usr/bin/{executable_name}", log_output=f"{executable_name}.log", service_args=['--log-no-truncate'])
```